### PR TITLE
[develop] Trello-LusVHJdO: Add user avatars

### PR DESCRIPTION
**Pull request by wildeastengineer**
[Trello-LusVHJdO](https://trello.com/c/LusVHJdO): Add user avatars

---

selection_001

### CODE REVIEW

File: client/components/common/Avatar/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+if (process.env.BROWSER) {
+    require('./avatar.scss');
+}
+
+const propTypes = {
+    id: PropTypes.string
+};
+
+const defaultProps = {
+    id: 'default-1'
```
Comment (AndreySurzhan) [inline client/components/common/Avatar/Avatar.jsx]: @wildeastengineer do we have something unisex for default one?
If you still think that that avatar is ok for default you can go head and merge this PR

Comment (wildeastengineer) [inline client/components/common/Avatar/Avatar.jsx]: @AndreySurzhan unfortunately don't. But I think it's ok to show to show first male avatar and then let user choose what he|she want. 
Nevertheless I've renamed avatar's files to separate male from female ones.
Is it ok for you?

**Comment by wildeastengineer** [comment]
@AndreySurzhan what do you think about this:

selection_002

**Comment by AndreySurzhan** [comment]
@wildeastengineer :) looks overunisex)) I'd rather leav eyes and mouth )

**Pull request merged**